recipes/libpcsclite/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "libpcsclite" %}
+{% set version = "1.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pcsclite.apdu.fr/files/pcsc-lite-{{ version }}.tar.bz2
+  sha256: cbcc3b34c61f53291cecc0d831423c94d437b188eb2b97b7febc08de1c914e8a
+
+build:
+  skip: True  # [not linux]
+  script:
+    - set -e
+    - set -x
+    - if test -f Makefile; then make distclean; fi
+    - rm -rf *.cache *.m4 config.guess config.log config.status config.sub depcomp ltmain.sh
+    - autoreconf --verbose --install --force
+    - ./configure --prefix=$PREFIX
+    - make install
+    - make distcheck
+    - mv $PREFIX/sbin/* $PREFIX/bin/
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libpcsclite') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ cdt('systemd-devel') }}
+    - autoconf-archive
+    - automake
+    - flex
+    - libtool
+    - make
+    - pkg-config 
+    - sysroot_linux-64 2.17  # [linux64]
```

I noticed `configure` tried to call `/usr/bin/file` and failed:
```suggestion
    - file
    - sysroot_linux-64 2.17  # [linux64]
```